redaxo/src/core/schemas/config.json

```
@@ -0,0 +1,375 @@
+            },
+            "additionalProperties": false,
+            "definitions": {
+                "min_max": {
+                    "type": "object",
+                    "properties": {
+                        "min": {
+                            "type": "integer",
+                            "minimum": 0
+                        },
+                        "max": {
+                            "type": "integer",
+                            "minimum": 0
+                        }
+                    },
+                    "additionalProperties": false,
+                    "minProperties": 1
+                },
+                "interval": {
+                    "type": "string",
+                    "pattern": "^P(\\d+Y)?(\\d+M)?(\\d+W)?(\\d+D)?(T(\\d+H)?(\\d+M)?(\\d+S)?)?$"
+                }
+            }
+        },
+        "lang": {
+            "description": "Default language",
+            "$ref": "#/definitions/lang"
+        },
+        "lang_fallback": {
+            "description": "Fallback languages for missing translations",
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/lang"
+            }
+        },
+        "use_https": {
+            "description": "Force https",
+            "$ref": "#/definitions/bool-or-environment"
+        },
+        "use_hsts": {
+            "description": "Set HSTS header (HTTP Strict Transport Security)",
+            "type": "boolean"
+        },
+        "hsts_max_age": {
+            "description": "Max age for HSTS header (seconds)",
+            "type": "integer"
+        },
+        "use_gzip": {
+            "description": "Use gzip encoding",
+            "$ref": "#/definitions/bool-or-environment"
+        },
+        "use_etag": {
+            "description": "Use Etag header",
+            "$ref": "#/definitions/bool-or-environment"
+        },
+        "use_last_modified": {
+            "description": "Use Last-Modified header (not recommended)",
+            "$ref": "#/definitions/bool-or-environment"
+        },
+        "start_page": {
+            "description": "Default start page for backend",
+            "type": "string"
+        },
+        "timezone": {
+            "description": "Timezone (e.g. Europe/Berlin)",
+            "type": "string"
+        },
+        "socket_proxy": {
+            "description": "Proxy used for all rex_socket connections",
+            "type": ["null", "string"]
+        },
+        "setup_addons": {
+            "description": "Addons loaded in setup and in safe mode",
+            "type": "array",
+            "items": {
+                "type": "string",
+                "pattern": "^[^/]*$"
+            }
+        },
+        "system_addons": {
+            "description": "Addons marked as system addons (installed in setup and not deletable)",
+            "type": "array",
+            "items": {
+                "type": "string",
+                "pattern": "^[^/]*$"
```

Leer nicht erlauben?
```suggestion
                "pattern": "^[^/]+$"
```